### ISSUE TITLE
[BUG]: Some images are missing from the PDF export of arneis.readthedocs.io

### ISSUE THREAD

**Issue by gmacario** [B-AROL-O/ARNEIS#96]
### Description of the bug

Examples:

* All the images in section "Implement preview for Pull Requests" of https://github.com/B-AROL-O/ARNEIS/blob/main/docs/howto/howto-configure-arneis-readthedocs-io.md do not show up in the generated PDF

### Steps To Reproduce

1. Go to https://arneis.readthedocs.io/_/downloads/en/latest/pdf/
2. Scroll up to the end of Page 8 (Section 2.1.6)

### Expected result

Something similar to what displayed in HTML at https://arneis.readthedocs.io/en/latest/howto/howto-configure-arneis-readthedocs-io.html#implement-preview-for-pull-requests

![image](https://user-images.githubusercontent.com/75182/151246638-c90e7f03-e62d-454c-8478-e1df6b8bceac.png)

### Actual result

![image](https://user-images.githubusercontent.com/75182/151246154-a05e1dbf-2b04-49c1-90ac-7eb14cae6977.png)

**Comment by gmacario** [comment]
Root cause has to be investigated, but I tend to suspect that those images are missing since they are not part of the source tree.

Please have a look at the src URL of the `<img>` elements in https://github.com/B-AROL-O/ARNEIS/blob/main/docs/howto/howto-configure-arneis-readthedocs-io.md?plain=1#L209-L229

```markdown
### Implement preview for Pull Requests

<!-- (2022-01-25 05:35 CET) -->

References:
* <https://github.com/B-AROL-O/ARNEIS/issues/85>
* [Preview Documentation from Pull Requests](https://docs.readthedocs.io/page/guides/autobuild-docs-for-pull-requests.html)

In order to enable the Pull Request preview feature you should execute the following steps:

1. In your _Read the Docs_ project page select the `Admin` menu

   <img width="846" alt="Schermata 2022-01-25 alle 18 36 42" src="https://user-images.githubusercontent.com/25162080/151029713-e7ef6360-e1b7-4f0f-9a8c-0ec154701581.png">

1. In `Advanced Settings` select **Build pull requests for this project**

   <img width="829" alt="Schermata 2022-01-25 alle 18 40 42" src="https://user-images.githubusercontent.com/25162080/151030087-5fa2549e-2235-4e86-83ad-be31bb1f3294.png">

3. That's all. For each pull request an automatic check which builds the document on _Read The Docs_ is added. You can check the result of this compilation in the `Detail` information for this check.

   <img width="730" alt="Schermata 2022-01-25 alle 18 52 38" src="https://user-images.githubusercontent.com/25162080/151031987-2e64c678-0638-43a9-90c0-f965b2ede3c4.png">
```

**Comment by gmacario** [comment]
Postponing to next sprint since resolution is blocked by #191

**Comment by gmacario** [comment]
Unassigning myself, moving to dev-cw16.

**Comment by alv67** [comment]
assigned to me but moving to dev-cw17
